docker/autoware-auto-packages.txt

```diff
@@ -29,3 +29,8 @@ trajectory_follower
 velodyne_driver
 voxel_grid
 voxel_grid_nodes
+tracking
+tracking_nodes
```

Comment:
For tracking and tracking_nodes to be built, we need following:
measurement_conversion
hungarian_assigner
mpark_variant_vendor

None are nodes to be tested, but remember to remove them from the ignore list